test/functional/feature_init.py

```diff
@@ -124,18 +125,31 @@ def check_clean_start():
             check_clean_start()
             self.stop_node(0)
 
+        self.log.info("Test startup errors after perturbing certain essential files")
         for file_patt, err_fragment in files_to_perturb.items():
+            shutil.copytree(node.chain_path / "blocks", node.chain_path / "blocks_bak")
+            shutil.copytree(node.chain_path / "chainstate", node.chain_path / "chainstate_bak")
             target_files = list(node.chain_path.glob(file_patt))
 
             for target_file in target_files:
                 self.log.info(f"Perturbing file to ensure failure {target_file}")
-                with open(target_file, "rb") as tf_read, open(target_file, "wb") as tf_write:
+                with open(target_file, "rb") as tf_read:
```

Comment:
I guess I don't see why we are reading the content if we don't care about it anyway here. I think we can just override it and that simplifies things. I drafted that in this commit: https://github.com/fjahr/bitcoin/commit/84132abe9bd13845ef611615a30e53fbaffafb05

Reply:
I agree, that is simpler!